e2e/app/page/admin-user-audit-page.ts

```diff
@@ -38,8 +39,9 @@ export default class UserAuditPage extends AuthenticatedPage {
     return usernameValue;
   }
 
-  async clickUserAdminLink(): Promise<void> {
+  async clickUserAdminLink(): Promise<UserProfileAdminPage> {
     const link = new Link(this.page, '//a[text()="User"]');
     await link.click();
+    return new UserProfileAdminPage(this.page);
```

Comment:
Good addition. Even better if it's `return new UserProfileAdminPage(this.page).waitForLoad();`